pyCFOFiSAX/_tree_iSAX.py

```diff
@@ -314,10 +314,10 @@ def preprocessing_for_icfof(self, ntss_tmp, bool_print: bool = False, count_num_
 
     def _minmax_nodes(self):
         """
-        Retourne les bornes des nœuds de l'arbre.
-        Fait appel à :func:`~pyCFOFiSAX.node.RootNode._do_bkpt`.
+        Returns the terminals of the nodes of the shaft.
```

Comment:
Returns the breakpoints of the nodes of the tree

Reply:
```suggestion
        Returns the breakpoints of the nodes of the tree.
```